src/talk/renderer/notifications/notifications.store.js

```diff
@@ -355,3 +378,14 @@ export function createNotificationStore() {
 }
 
 export const notificationsStore = createNotificationStore()
+
+subscribeBroadcast('notifications:missedCall', ({ token, name, avatar }) => {
+	const notification = new Notification(t('talk_desktop', 'You have missed a call from {name}', { name }), {
```

Comment:
Align with what Talk backend sends? Would be easier to translate (or even copy-paste today from spreed repo)
```suggestion
	const notification = new Notification(t('talk_desktop', 'You missed a call from {user}' { user: name }), {
```

Reply:
> You have missed a call from {name}

Does not work well in other languages. The notification should also give you the conversation type, and then we can toggle between:
- https://github.com/nextcloud/spreed/blob/main/lib/Notification/Notifier.php#L1021 and
- https://github.com/nextcloud/spreed/blob/main/lib/Notification/Notifier.php#L1054